src/Elastic.Apm/Api/ApiConstants.cs

```diff
@@ -29,5 +29,9 @@ public struct ApiConstants
 		public const string TypeStorage = "storage";
 		public const string TypeApp = "app";
 		public const string TypeUnknown = "unknown";
+
+		public const string ActivationMethodNuGet = "nuget";
```

Comment:
I'd move this to another place. The `ApiConstants` was meant to hold things that you pass to the public API - specifically transaction/span type/subtype/action. 

I think activation method is a bit different. I'd not even make these values public.

Reply:
Yes, makes sense. I'll move these constants somewhere else and tighten their visibility.